packages/react-native/src/evm/wallets/connectors/embedded-wallet/embedded/auth.ts

```diff
@@ -42,12 +42,24 @@ import {
 import { InAppBrowser } from "react-native-inappbrowser-reborn";
 import { createErrorMessage } from "./helpers/errors";
 import {
-  appBundleId,
-  reactNativePackageVersion,
-} from "../../../../utils/version";
-import { BUNDLE_ID_HEADER } from "../../../../constants/headers";
+  BUNDLE_ID_HEADER,
+  X_SDK_NAME_HEADER,
+  X_SDK_OS_HEADER,
+  X_SDK_PLATFORM_HEADER,
+  X_SDK_VERSION_HEADER,
+} from "../../../../constants/headers";
 import { ANALYTICS } from "./helpers/analytics";
 
+const HEADERS = {
+  [EWS_VERSION_HEADER]: (globalThis as any).X_SDK_VERSION,
+  [BUNDLE_ID_HEADER]: (globalThis as any).APP_BUNDLE_ID,
+  [THIRDWEB_SESSION_NONCE_HEADER]: ANALYTICS.nonce,
+  [X_SDK_NAME_HEADER]: (globalThis as any).X_SDK_NAME,
+  [X_SDK_OS_HEADER]: (globalThis as any).X_SDK_OS,
+  [X_SDK_PLATFORM_HEADER]: (globalThis as any).X_SDK_PLATFORM,
+  [X_SDK_VERSION_HEADER]: (globalThis as any).X_SDK_VERSION,
```

Comment:
are you sure this is ok? not sure if the values are set (in the provider) at the time of importing this file